spec/lib/appsignal/event_formatter/active_record/sql_formatter_spec.rb

```diff
@@ -114,12 +114,6 @@
 
             it { should == ['Model load', "\"id\" = ?"] }
           end
-
```

Comment:
This shouldn't happen in this case because it's a SQL style where the table names are in double quotes and the values in single quotes.